src/renderer/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -127,7 +116,7 @@ const Dashboard: React.FunctionComponent<IOwnProps & IInjectedProps & Pick<IRoot
         }
 
         loadValidators();
-    },[]);
+    },[props.auth.account && props.auth.account.numberOfValidators]);
```

Comment:
Cant you do here just `props.auth.account && props.auth.account.validators.length`?

Reply:
Nop, can't access validators

Reply:
props.auth.account.getValidators().length? :)

Reply:
lol, that actually works xD